source/deprecations/v3.x.html.md

```diff
@@ -53,3 +53,42 @@ object.notifyPropertyChange('someProperty');
 ##### id: ember-metal.getting-each
 
 Calling `array.get('@each')` is deprecated. `@each` may only be used as dependency key.
+
+#### Use console rather than Ember.Logger
```

Comment:
This deprecation should be under "upcoming deprecation" as it is not included in a release as of yet :)